controller/array_action/array_mediator_ds8k.py

```diff
@@ -427,24 +429,21 @@ def _get_api_volume_by_name(self, volume_name, pool_id):
                 "pool_id is not specified, can not get volumes from storage."
             )
             raise array_errors.PoolParameterIsMissing(self.array_type)
```

Comment:
if this is raised here, then why bother raising it in other methods as well? (e.g. `get_volume`, `get_snapshot`, etc...)

Reply:
done (remove from other places that using this method)